volatility3/framework/layers/crash.py

```diff
@@ -28,6 +28,9 @@ class WindowsCrashDump32Layer(segmented.SegmentedLayer):
     SIGNATURE = 0x45474150
     VALIDDUMP = 0x504D5544
 
+    DUMP_TYPE_FULL = 1
```

Comment:
Crash info should probably make use of these constants (at the moment it hardcodes the values).